pyproject.toml

```diff
@@ -38,7 +38,9 @@ remote = [
     "aiohttp",
     "s3fs",
 ]
-
+obstore = [
+    "obstore @ git+https://github.com/developmentseed/obstore@main#subdirectory=obstore",
+]
```

Comment:
The `ObjectStore` implementation was just merged into zarr-python! Does that make any difference for us here? Or  should we continue to use the `obstore` package directly?

Reply:
@kylebarron would you be willing to take a quick look at https://github.com/zarr-developers/VirtualiZarr/blob/28255e7ddf1c39a55d94e792540a48d944c115c7/virtualizarr/manifests/store.py#L169-L337 and provide your opinion on whether I should be subclassing zarr's ObjectStore rather than building this separately? I'm leaning towards this being distinct enough to warrant its own implementation, but would appreciate a second opinion.

Reply:
> The `ObjectStore` implementation was just merged into zarr-python! Does that make any difference for us here? Or should we continue to use the `obstore` package directly?

You should definitely use `obstore` from PyPI instead of from a git hash, regardless of whether you subclass the zarr store or not.

Reply:
And given that you manage multiple stores internally, I think it's fine to keep this as a separate implementation

Reply:
Do you think we should pin the specific version (e.g., v0.6.0)? I.e., do you expect we'll encounter breaking changing in the features we're using here?

Reply:
I don't think it should be pinned. Perhaps a lower bound of 0.5 or 0.5.1, but not an upper bound.

 I don't expect any breaking changes, especially to the methods for actually accessing data.